pkgs/by-name/do/doas-keepenv/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  bash,
+  coreutils,
+  doas,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
```

No need for rec, nothing is self-referential. Anyways, nowadays using finalAttrs is best practices.
```suggestion
stdenv.mkDerivation {
```